modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase

```diff
@@ -100,7 +100,7 @@ definition {
 	@description = "LPS-162894 - Verify that is not possible to submit the form without fill all required fields"
 	@priority = 3
 	test CannotSubmitFormWithoutFillRequiredField {
-		task ("When the user submit an organization form without a required field filled in") {
+		task ("When the Employee user submit an organization form without a required field filled in") {
```

Comment:
Just a typo here, could you change this task to: `"When the Employee user submits an organization form without a required field filled in"`